From 9312e7898b6863fb8d472b854a3b9d05e0cfdcbc Mon Sep 17 00:00:00 2001
From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Fri, 16 Jul 2010 15:18:02 +0200
Subject: [PATCH] loader: when creating sdksyms.c only include shmint.h if MITSHM is enabled #29109

When configured with --disable-mitshm the symbols declared in shmint.h
do not exist. By guarding the include with '#ifdef MITSHM' these
symbols are skipped when generating sdksyms.c with --disable-mitshm.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 hw/xfree86/loader/sdksyms.sh |    2 ++
 1 files changed, 2 insertions(+), 0 deletions(-)

diff --git a/hw/xfree86/loader/sdksyms.sh b/hw/xfree86/loader/sdksyms.sh
index 13c5ae5..bbda4fa 100755
--- a/hw/xfree86/loader/sdksyms.sh
+++ b/hw/xfree86/loader/sdksyms.sh
@@ -50,7 +50,9 @@ cat > sdksyms.c << EOF
  */
 #include "geext.h"
 #include "geint.h"
+#ifdef MITSHM
 #include "shmint.h"
+#endif
 #if XINERAMA
 # include "panoramiXsrv.h"
 # include "panoramiX.h"
-- 
1.7.1

